erpnext/manufacturing/doctype/job_card/job_card.py

```diff
@@ -522,23 +526,50 @@ def get_current_operation_data(self):
 			},
 		)
 
-	def set_transferred_qty_in_job_card(self, ste_doc):
+	def set_transferred_qty_in_job_card_item(self, ste_doc):
```

Comment:
also, if possible lets avoid renaming public function 😅

Reply:
had to be done at some point. The function names almost clash and are so confusing, cant understand the purpose.
There is `set_transferred_qty` and `set_transferred_qty_in_job_card`. Both set transferred qty in job card, now guess how they are different 🥲